Microsoft.Toolkit.Uwp.Connectivity/BluetoothLEHelper/ObservableBluetoothLEDevice.cs

```diff
@@ -134,17 +135,25 @@ public int Compare(object x, object y)
         private ObservableCollection<ObservableGattDeviceService> _services =
             new ObservableCollection<ObservableGattDeviceService>();
 
+        /// <summary>
+        /// Gets or sets which DispatcherQueue is used to dispatch UI updates.
+        /// </summary>
+        public DispatcherQueue DispatcherQueue { get; set; }
```

Comment:
Why does this need to be public?

Reply:
We didn't have the requirement that this class needed to be instantiated from a UI thread. Since this might be a scenario for some users, I wanted to make sure that they could set the dispatcher if they already have an instance of it (thru the ctor), or change it later (thru the public property).